### DISCUSSION TITLE
Is a 100% screen-to-body ratio fullscreen available in brower?

### DISCUSSION THREAD

**Discussion by RunguangLi** [touying-typ/touying#102] [Q&A] [answered]
Hi, thanks for the work! I really enjoy using Touying for my presentations. I noticed that the fullscreen mode by pressing F11 in the browser doesn't seem to be fullscreen at 100% screen-to-body ratio. Is there a way to remove the top line to display as if it was a simple PDF in full-screen mode? Thanks
Runguang
browser fullscreen 
![1729726449480](https://github.com/user-attachments/assets/553954eb-e2dd-4d64-9f90-c06f5a53ee69)
PDF fullscreen
![1729727179638](https://github.com/user-attachments/assets/24919987-dc2d-4401-bd4e-edea46c9214b)

**Comment by OrangeX4** [comment, marked as answer]
I think it depends on the presentation viewer you use. Why not just show the pdf with pdf viewer?
